tgui/packages/tgui/styles/themes/hackerman.scss

```diff
@@ -51,6 +51,11 @@
     border-style: outset;
     border-color: #00AA00;
     outline: base.em(1px) solid rgb(0, 122, 0);
+    box-sizing: content-box;
```

Comment:
I don't know why exactly, but this is the line that makes dradis circles fat

Reply:
Is the dradis using the hackerman UI theme? I did this to include the border in the total size of the box, so buttons would fit within their specific width and height rather than expand out of that, as it was making hackerman UIs look bloated.

Reply:
> Is the dradis using the hackerman UI theme? I did this to include the border in the total size of the box, so buttons would fit within their specific width and height rather than expand out of that, as it was making hackerman UIs look bloated.

Yes the dradis is using that thene

Reply:
It really shouldn't be on those buttons specifically. Give them some custom css to set it back to content-box